crates/sui-sdk/README.md

```diff
@@ -1,6 +1,134 @@
+
+```rust
+use sui_sdk::SuiClientBuilder;
+
+#[tokio::main]
+async fn main() -> Result<(), anyhow::Error> {
+    // Sui devnet -- https://fullnode.devnet.sui.io:443
+    let sui_devnet = SuiClientBuilder::default().build_devnet().await?;
+    println!("Sui devnet version: {}", sui_devnet.api_version());
+
+    // Sui testnet -- https://fullnode.testnet.sui.io:443
+    let sui_testnet = SuiClientBuilder::default().build_testnet().await?;
+    println!("Sui testnet version: {}", sui_testnet.api_version());
+
+    Ok(())
+}
+
+```
+
+
+
 # Rust SDK examples
 
-This directory contains examples of interacting with a Move language smart contract using the Sui Rust SDK. See the [introduction to the Rust SDK](https://github.com/MystenLabs/sui/blob/main/doc/src/build/rust-sdk.md) for additional details.
+The [examples](https://github.com/MystenLabs/sui/tree/main/crates/sui-sdk/examples) folder provides both basic and advanced examples.
+
+There are five files ending in `_api.rs` which provide code examples of the corresponding APIs and their methods. These showcase how to use the Sui Rust SDK, and can run against the Sui testnet. Below are instructions on the prerequisites and how to run these examples.
+
+## Preqrequisites
+
+Unless otherwise specified, most of these examples assume `Rust` and `cargo` are installed, and that there is an available internet connection. The examples connect to the Sui testnet (`https://fullnode.testnet.sui.io:443`) and execute different APIs. 
+
+## Running the examples
+
+In the root folder of the `sui` repository (or in the `sui-sdk` crate folder), you can individually run examples using the command  `cargo run --example filename` (without `.rs` extension). For example:
+* `cargo run --example sui_client` -- this one requires a local Sui network running (see [here](#Connecting to Sui Network
+)). If you do not have a local Sui network running, please skip this example.
+* `cargo run --example coin_read_api`
+* `cargo run --example event_api`
+* `cargo run --example governance_api`
+* `cargo run --example read_api`
+* `cargo run --example transaction_builder_api`
+
+## Basic Examples
+
+### Connecting to Sui Network
+The `SuiClientBuilder` struct provides a connection to the JSON-RPC server that you use for all read-only operations. The default URLs to connect to the Sui network are:
+
+- Local: http://127.0.0.1:9000
+- Devnet: https://fullnode.devnet.sui.io:443
+- Testnet: https://fullnode.testnet.sui.io:443
+- Mainnet: https://fullnode.mainnet.sui.io:443
+
+For all available servers, see [here](https://sui.io/networkinfo). 
+
+For running a local Sui network, please follow [this guide](https://docs.sui.io/build/sui-local-network) for installing Sui and [this guide](https://docs.sui.io/build/sui-local-network#start-the-local-network) for starting the local Sui network. 
+
+
+```rust
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
nit: remove the license in the code snippet? You have one for the README file overall.